From: Miaohe Lin <linmiaohe@huawei.com>
Subject: mm/memcg: remove definition of MEM_CGROUP_ID_MAX when !CONFIG_MEMCG
Date: Thu, 6 Jul 2023 19:28:20 +0800

MEM_CGROUP_ID_MAX is only used when CONFIG_MEMCG is configured.  Remove
unneeded !CONFIG_MEMCG variant.

Link: https://lkml.kernel.org/r/20230706112820.2393447-1-linmiaohe@huawei.com
Signed-off-by: Miaohe Lin <linmiaohe@huawei.com>
Reviewed-by: Muchun Song <songmuchun@bytedance.com>
Cc: Johannes Weiner <hannes@cmpxchg.org>
Cc: Michal Hocko <mhocko@kernel.org>
Cc: Roman Gushchin <roman.gushchin@linux.dev>
Cc: Shakeel Butt <shakeelb@google.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/linux/memcontrol.h |    1 -
 1 file changed, 1 deletion(-)

--- a/include/linux/memcontrol.h~mm-memcg-remove-definition-of-mem_cgroup_id_max-when-config_memcg
+++ a/include/linux/memcontrol.h
@@ -1158,7 +1158,6 @@ unsigned long mem_cgroup_soft_limit_recl
 #else /* CONFIG_MEMCG */
 
 #define MEM_CGROUP_ID_SHIFT	0
-#define MEM_CGROUP_ID_MAX	0
 
 static inline struct mem_cgroup *folio_memcg(struct folio *folio)
 {
_
